release-notes/9.0/patch-release-index.json

```diff
@@ -0,0 +1,41 @@
+{
+  "channel-version": "9.0",
+  "releases": [
+    {
+      "release-version": "9.0.0-preview.6",
+      "release-date": "2024-07-09",
+      "security": false,
+      "release.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata//9.0/preview/preview6/release.json"
```

Comment:
Just thinking out loud - would it be better to name this something as to what it is, rather than just the name of the file (`release-info-uri` or something)?

Reply:
Great topic. I am not a fan of the current naming scheme. I just copied what was already there for consistency.

Example: https://github.com/dotnet/core/blob/a6569a1037f9405b678b30c03ece7df475c04c55/release-notes/releases-index.json#L16

An argument could be made that keeping each file consistent makes sense, but that this is a new file so an opportunity to do better. Compelling?

Reply:
I can see both arguments - consistency is nice, but holding back improvement for the sake of it isn't great either. I could go either way 😄

Reply:
I'll say that as a consumer of this file having property keys with `.` in them is a _giant pain_.

Reply:
How does that show up @baronfel? 

We could also come up with a new scheme, enable those everywhere and keep the only properties for compat.

Reply:
For .NET, when defining binding models for this property you have to annotate the property with the JsonPropertyName attribute, that's the main pain point. Some bindings in other libraries or tools (like `jq` path patterns) are also tricky to encode compared to property keys with dashes or underscores.

Reply:
OK. I'm with you. I'm convinced. I'll address this. I'm very much not a fan of this naming scheme.